app/controllers/application_controller.rb

```diff
@@ -5,6 +5,8 @@ class ApplicationController < ActionController::Base
   before_action :set_system_setting
   around_action :switch_locale
 
+  include Pagy::Backend
+
```

Comment:
I try not to include much in `ApplicationController` (or `ApplicationHelper`). It's so easy for these base classes to quickly become 'kitchen-sink's of clutter, all of which is inherited by all controllers.

I realize that is what's recommended in the `pagy` docs and also pretty common in the rails world, so definitely not a critique of your work!

If you're open to trying some alternatives:
* Call `Pagy::Backend.pagy` directly in actions. I _think_ that should work and is very explicit and straightforward. `Pagy::Frontend.pagy_nav` in the view. On the downside, it's more wordy :grin:
* Include the backend and frontend only in controllers that need them (+ a `helper` call might also be necessary?).
* Extract a thin module/concern to reduce the duplication of `include`ing the pagy modules in controllers 

Curious your thoughts on this?

Reply:
I am open to calling `Pagy::Backend.pagy` explicitly, that's no problem. I originally had the `include` only in the `PeopleController` but thought that more pagination was likely coming soon so I'd get ahead of it (a pre-mature optimization on my part). 

I'll see if the backend and frontend direct calls work the same and then update if they do.

Reply:
Looks like calling the `pagy` method directly on the `Pagy::Bakend` module doesn't work, we have to include it. I'll move the include to the `people_controller` instead. Same thing for `Pagy::Frontend`, I'll move that to a new `people_helper` to reduce the impact on the rest of the app.